src/test/java/uk/ac/ebi/atlas/home/HomeControllerTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.ac.ebi.atlas.home;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
+
+@ExtendWith(SpringExtension.class)
+@WebAppConfiguration
+@ContextConfiguration(classes = TestConfig.class)
+class HomeControllerTest {
+
+    @Autowired
+    private WebApplicationContext wac;
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    void setUp() {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
+    }
+
+    @Test
+    void homeReturnsValidModel() throws Exception {
+        this.mockMvc.perform(get("/home"))
+                .andExpect(status().isOk())
+                .andExpect(view().name("home"));
+    }
+
+    @Test
+    void homeModelHaveAssayInfo() throws Exception {
+        this.mockMvc.perform(get("/home"))
+                .andExpect(status().isOk())
+                .andExpect(view().name("home"))
```

Comment:
The view test is already part of the previous one. You don’t need to check twice.